relay-server/src/actors/store.rs

```diff
@@ -836,17 +840,40 @@ impl StoreService {
                     error = &error as &dyn std::error::Error,
                     "failed to parse span"
                 );
+                self.outcome_aggregator.send(TrackOutcome {
+                    category: DataCategory::SpanIndexed,
+                    event_id,
+                    outcome: Outcome::Invalid(DiscardReason::InvalidSpan),
+                    quantity: 1,
+                    remote_addr,
+                    scoping,
+                    timestamp: instant_to_date_time(start_time),
+                });
                 return Ok(());
             }
         };
 
         span.duration_ms = ((span.end_timestamp - span.start_timestamp) * 1e3) as u32;
         span.event_id = event_id;
-        span.project_id = project_id.value();
+        span.project_id = scoping.project_id.value();
         span.retention_days = retention_days;
         span.start_timestamp_ms = (span.start_timestamp * 1e3) as u64;
 
-        self.produce(KafkaTopic::Spans, organization_id, KafkaMessage::Span(span))?;
+        self.produce(
+            KafkaTopic::Spans,
+            scoping.organization_id,
+            KafkaMessage::Span(span),
+        )?;
+
+        self.outcome_aggregator.send(TrackOutcome {
+            category: DataCategory::SpanIndexed,
+            event_id,
+            outcome: Outcome::Accepted,
+            quantity: 1,
+            remote_addr,
```

Comment:
Let's set these fields (`event_id` and `remote_addr`) to `None` so that the outcome aggregator can summarize outcomes. It already drops these fields under certain conditions (e.g. `Invalid` outcome), but spans are very high volume so we don't want to ever generate a separate outcome kafka message for every span.

```suggestion
            event_id: None,
            outcome: Outcome::Accepted,
            quantity: 1,
            remote_addr: None,
```

Reply:
Actually, could you also add `Accepted` to the `matches!` condition here, so that we don't mess this up in any future use cases of accepted? Thanks! https://github.com/getsentry/relay/blob/5cd6c1ed620f0ae6cf5823b98245c32582d45f60/relay-server/src/actors/outcome_aggregator.rs#L163-L172